llvm/lib/Object/ELF.cpp

```diff
@@ -839,7 +840,6 @@ decodeBBAddrMapImpl(const ELFFile<ELFT> &EF,
       NumBlocksInBBRange = readULEB128As<uint32_t>(Data, Cur, ULEBSizeErr);
     }
     std::vector<BBAddrMap::BBRangeEntry> BBRangeEntries;
-    uint32_t TotalNumBlocks = 0;
```

Comment:
I don't see why this line has been moved.

Reply:
Good catch, I must have mismatched the branches..fixed.